akd/src/ecvrf/no_vrf.rs

```diff
@@ -0,0 +1,123 @@
+// Copyright (c) Facebook, Inc. and its affiliates.
+//
+// This source code is licensed under both the MIT license found in the
+// LICENSE-MIT file in the root directory of this source tree and the Apache
+// License, Version 2.0 found in the LICENSE-APACHE file in the root directory
+// of this source tree.
+
+//! This module contains mock usage of the VRF functionality should someone choose they
```

Comment:
If we are not using it, let's consider adding it in another PR. It also has incomplete implementations.

Reply:
It's a crate-level feature, if someone does NOT want VRF functionality as it's a computation hit. I'll enable the test suite without VRFs in the CI pipeline, but the implementation _is_ complete. It's supposed to return an empty private key struct, essentially saying "there is no VRF key, and everything computed will be without VRFs"